clang/tools/clang-linker-wrapper/LinkerWrapperOpts.td

```diff
@@ -163,6 +163,15 @@ def sycl_post_link_options_EQ : Joined<["--", "-"], "sycl-post-link-options=">,
   Flags<[WrapperOnlyOption]>,
   HelpText<"Options that will control sycl-post-link step">;
 
+def sycl_module_split_mode_EQ :
```

Comment:
I think we will need to add tests to make sure that the existing tests which use the sycl-post-link will run the same way when we use the split library.

Thanks